internal/cmd/plugin/command_install.go

```diff
@@ -107,14 +101,14 @@ func installPlugin(manifest *Manifest, repositoryDir, installDir string) error {
 	language, ver := getLanguage(manifest)
 
 	switch language {
-	case Go:
+	case "go":
 		checkGoVersion(ver)
 		return installGoPlugin(manifest.Name)
-	case Python:
+	case "python":
 		checkPythonVersion(ver)
-		return installSimplePlugin(manifest.Name, repositoryDir, installDir, Python)
-	case Shell:
-		return installSimplePlugin(manifest.Name, repositoryDir, installDir, Shell)
+		return installSimplePlugin(manifest.Name, repositoryDir, installDir, "python")
+	case "shell":
+		return installSimplePlugin(manifest.Name, repositoryDir, installDir, "shell")
```

Comment:
GitHub won't let me suggest a change here, but we can combine these into one, and pass `language` as the fourth parameter.

Reply:
I set it up this way because the Python case has an additional call to `checkPythonVersion`.

Reply:
If we take the interface route, maybe we can call `CheckVersion()` followed by `InstallPlugin()` for each case here, then we won't have to maintain this `switch` statement for future languages.

Reply:
Good point, I like that. I'll go ahead and make a version of this.

Reply:
Hmm, well I kept the switch because I still need to pick out the right interface implementations, but at least it's a simpler switch now... what do you think?